arviz/data/io_pyro.py

```diff
@@ -108,6 +108,7 @@ def arbitrary_element(dct):
 
         observations = {}
         if self.model is not None:
+            # pylint: disable=not-callable
```

Comment:
this'll disable the `not-callable` check everywhere in the file below this line - I presume the intention was just to disable it for `pyro.poutine.trace`? If so, perhaps
```suggestion
            # pylint: disable=not-callable
            trace = pyro.poutine.trace(self.model).get_trace(*self._args, **self._kwargs)
            # pylint: enable=not-callable
```
or
```suggestion
            trace = pyro.poutine.trace(self.model).get_trace(*self._args, **self._kwargs)  # pylint: disable=not-callable
```
would be better?

Reply:
Ohh thanks, Ill change this